revisions-cli.php

```diff
@@ -111,6 +114,7 @@ public function list_( $args = array(), $assoc_args = array() ) {
 		}
 
 		$formatter = new \WP_CLI\Formatter( $assoc_args, array( 'ID', 'post_parent', 'post_title' ), 'revisions' );
+		$formatter->format = ( in_array( $assoc_args['format'], [ 'csv', 'json', 'table' ], true ) ) ? $assoc_args['format'] : 'table';
```

Comment:
I think this is set automatically by the line above. Also, throws an unset index warning if `--format` isn't passed.

Testing suggests this line isn't needed.

Reply:
Ah yes, I see.